mindsdb/api/mysql/mysql_proxy/executor/executor_commands.py

```diff
@@ -1147,13 +1154,9 @@ def answer_drop_view(self, statement):
     @mark_process('learn')
     def answer_create_predictor(self, statement):
         integration_name = self.session.database
-        if len(statement.name.parts) > 1:
```

Comment:
this piece of code allows to create model in other project. for example active project is mindnsdb but we can call 'create mode proj1.mindsdb'

Reply:
Ahhh I see, will restore!

Reply:
Hmm, looks like it is not being used, according to the linter 🤔